scripts/staging-smoke.sh

```
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Einfache Smoke-Tests für Staging
+# Nutzung: export STAGING_URL=https://staging.example.com && export SERVICE_USER_TOKEN=... && ./scripts/staging-smoke.sh
+set -euo pipefail
+: ${STAGING_URL:?STAGING_URL muss gesetzt sein}
+
+REQID_PREFIX="smoke-$(date +%s)"
+
+echo "1) Health check"
+curl -fsS -i -H "X-Request-ID: ${REQID_PREFIX}-health" "${STAGING_URL}/api/health" || { echo "Health check failed"; exit 2; }
+
+echo "\n2) Service: Liste Kurse (Service-User)"
```

Shellcheck (0.11.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)